src/frontend-samples/explode-sample/ExplodeUI.tsx

```diff
@@ -0,0 +1,184 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelTileRpcInterface, TileVersionInfo } from "@bentley/imodeljs-common";
+import { IModelApp, IModelConnection, Viewport } from "@bentley/imodeljs-frontend";
+import { Button, Select, Slider } from "@bentley/ui-core";
+import "common/samples-common.scss";
+import { ControlPane } from "Components/ControlPane/ControlPane";
+import { ReloadableViewport } from "Components/Viewport/ReloadableViewport";
+import * as React from "react";
+import ExplodeApp from "./ExplodeApp";
+
+interface SampleProps {
+  iModelName: string;
+  iModelSelector: React.ReactNode;
+}
+
+interface ExplodeState {
+  isInit: boolean;
+  object: ExplodeObject;
+  explosionFactor: number;
+  emphasize: EmphasizeType;
+  tileVersion?: TileVersionInfo;
+  viewport?: Viewport;
+}
+
+interface ExplodeObject {
+  name: string;
+  elementIds: string[];
+}
+enum EmphasizeType {
+  None,
+  Emphasize,
+  Isolate,
+}
+
+function mapOptions(o: {}): {} {
+  const keys = Object.keys(o).filter((key: any) => isNaN(key));
+  return Object.assign({}, keys);
+}
+
+export default class ExplodeUI extends React.Component<SampleProps, ExplodeState> {
+  public state: ExplodeState;
+  private _objects: ExplodeObject[] = [
+    {
+      name: "Door",
```

Comment:
What is the point of exploding one element? It has no visible effect.

Reply:
Would be interesting to explode the exterior walls of the house to reveal the interior.

Reply:
> What is the point of exploding one element? It has no visible effect.

The door is only there for testing purposes.  Will remove before completion.  Added TODO: as to not forget.

> Would be interesting to explode the exterior walls of the house to reveal the interior.

Added the room with glass in the back of the house as an object.  Will consider others before completion.